deploy/.env

```diff
@@ -0,0 +1,13 @@
+REACT_APP_CONTRACTS_RELAY_HUB=0x66Fa9FEAfB8Db66Fe2160ca7aEAc7FC24e254387
+REACT_APP_CONTRACTS_DEPLOY_VERIFIER=0x5C6e96a84271AC19974C3e99d6c4bE4318BfE483
+REACT_APP_CONTRACTS_RELAY_VERIFIER=0x56ccdB6D312307Db7A4847c3Ea8Ce2449e9B79e9
+REACT_APP_CONTRACTS_SMART_WALLET_FACTORY=0xeaB5b9fA91aeFFaA9c33F9b33d12AB7088fa7f6f
+REACT_APP_CONTRACTS_SMART_WALLET=0xEdB6D515C2DB4F9C3C87D7f6Cefb260B3DEe8014
+REACT_APP_CONTRACTS_RIF_TOKEN=0xF5859303f76596dD558B438b18d0Ce0e1660F3ea
+REACT_APP_CONTRACTS_RELAY_WORKER=0x74105590d404df3f384a099c2e55135281ca6b40
+
+REACT_APP_RIF_RELAY_CHAIN_ID=31
+REACT_APP_RIF_RELAY_GAS_PRICE_FACTOR_PERCENT=0
+REACT_APP_RIF_RELAY_LOOKUP_WINDOW_BLOCKS=1e5
+REACT_APP_RIF_RELAY_PREFERRED_RELAYS=http://dev.relay.rifcomputing.net:8090
+
```

Comment:
I think that this file is related to the Dapp and not to the server. We could move it in the related repo, or keep it here, but if we decide for the latter, I'd specify it in the README section

Reply:
Erased this .env as you mention is not needed here